src/apps/gearbox/ethereum/gearbox.rewards.contract-position-fetcher.ts

```diff
@@ -0,0 +1,49 @@
+import { Inject } from '@nestjs/common';
+import { BigNumber, ethers } from 'ethers';
+
+import { IAppToolkit, APP_TOOLKIT } from '~app-toolkit/app-toolkit.interface';
+import { PositionTemplate } from '~app-toolkit/decorators/position-template.decorator';
+import { GetTokenBalancesParams } from '~position/template/contract-position.template.types';
+import { MerkleTemplateContractPositionFetcher } from '~position/template/merkle.template.contract-position-fetcher';
+import { Network } from '~types';
+
+import { AirdropDistributor, GearboxContractFactory } from '../contracts';
+
+import { EthereumGearboxRewardsMerkleCache, AIRDROP_DISTRIBUTOR, GEAR_TOKEN } from './gearbox.rewards.merkle-cache';
+
+@PositionTemplate()
+export class EthereumGearboxRewardsPositionFetcher extends MerkleTemplateContractPositionFetcher<AirdropDistributor> {
+  groupLabel = 'rewards';
+  merkleAddress = AIRDROP_DISTRIBUTOR;
+  network = Network.ETHEREUM_MAINNET;
+
+  isExcludedFromExplore = true;
+  isExcludedFromTvl = true;
+
+  constructor(
+    @Inject(APP_TOOLKIT) protected readonly appToolkit: IAppToolkit,
+    @Inject(GearboxContractFactory) protected readonly contractFactory: GearboxContractFactory,
+    @Inject(EthereumGearboxRewardsMerkleCache) private readonly merkleCache: EthereumGearboxRewardsMerkleCache,
+  ) {
+    super(appToolkit);
+  }
+
+  getContract(address: string): AirdropDistributor {
+    return this.contractFactory.airdropDistributor({ address, network: this.network });
+  }
+
+  async getRewardTokenAddresses() {
+    return [GEAR_TOKEN];
+  }
+
+  async getTokenBalancesPerPosition({ address, contract }: GetTokenBalancesParams<AirdropDistributor>) {
+    console.log({ address });
+    const rewardsData = await this.merkleCache.getClaim(GEAR_TOKEN, ethers.utils.getAddress(address));
+    console.log({ rewardsData });
```

Comment:
I don't think you ll need `ethers.utils.getAddress(address)` and `console.log`

Reply:
It needs to be checksummed address because in the merkle json files the keys are checksummed addresses